src/main/java/org/havenapp/main/PreferenceManager.java

```diff
@@ -152,7 +150,6 @@ public void setSignalUsername (String signalUsername)
      *
      * @return the verified Signal username; null when no Signal username is verified even though registered.
      */
-    @Nullable
```

Comment:
Hmm.. Is there any reason as to why we are removing this annotation?

Reply:
Hmm.  Can't remember now.  Must have thrown a lint error is my best guess (?)  I'll probably update this PR soon and take a look at that.

Reply:
Update reverted this change in https://github.com/guardianproject/haven/pull/405/commits/a857b347621772c1991bfb0a7f5901a9b0f3df37